code/go/0chain.net/smartcontract/zcnsc/authorizer.go

```diff
@@ -51,7 +52,7 @@ func (zcn *ZCNSmartContract) AddAuthorizer(tran *transaction.Transaction, inputD
 	// Decode input
 
 	params := AuthorizerParameter{}
-	err = params.Decode(inputData)
```

Comment:
We might want to add [stakePool settings](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/stakepool/stakepool.go#L33) to the input. These could then be validated against the bounds in the global node before creating the authoriser's stakePool.

Reply:
Done